services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy

```diff
@@ -128,30 +141,40 @@ class LinkPropertiesSpec extends BaseSpecification {
         }
     }
 
-    def "Updating costs via link props actually updates costs on ISLs"() {
-        given: "An ISL"
+    def "Updating cost and max bandwidth via link props actually updates cost and max bandwidth on ISLs"() {
+        given: "An active ISL"
         def isl = topology.islsForActiveSwitches.first()
+        def initialMaxBandwidth = islUtils.getIslInfo(isl).get().maxBandwidth
 
-        when: "Update cost on ISL via link props"
-        def cost = "12345"
-        def linkProps = [islUtils.toLinkProps(isl, ["cost": cost])]
+        when: "Update cost and max bandwidth on ISL via link props"
+        def value = "12345"
```

Comment:
I would've gone for different values for cost/bw to verify that they do not get mixed up during update

Reply:
Fixed.